packages/kokoas-client/src/pages/projRegisterV2/api/convertProjToForm.ts

```diff
@@ -181,6 +183,9 @@ export const convertProjToForm = ({
     storeName: storeName.value || cgStoreName.value,
     storeCode: storeCode.value || cgStoreCode.value,
     territory: (territory?.value as Territory || cgTerritory.value) || '',
+
+    // 管理台帳
+    ledgerInfo: ledgerInfo.value || '',
```

Comment:
```suggestion
    ledgerInfo: ledgerInfo.value || 'ANDPAD',
```

この段階で、空をデフォールトに変更していいかもしれません。

Reply:
ご指摘ありがとうございます。
react-hook-formの内部的な動きが分かっておらず、勉強になりました。
[0e34cc4](https://github.com/Lorenzras/yumecoco-monorepo/pull/1014/commits/0e34cc4f5412131988f0226cde4cd7bc1595121a)にて修正しましたので、ご確認をお願いいたします。